src/tools/tidy/src/cargo.rs

```diff
@@ -33,12 +34,22 @@ fn verify(tomlfile: &Path, libfile: &Path, bad: &mut bool) {
     let toml = t!(fs::read_to_string(&tomlfile));
     let librs = t!(fs::read_to_string(&libfile));
 
-    if toml.contains("name = \"bootstrap\"") {
-        return
-    }
-
     // "Poor man's TOML parser" -- just assume we use one syntax for now.
-    //
+    // 1. Check crate edition is 2018
+    // We just look for:
+    // ````
+    // edition = "2018"
+    // ````
+    // on a separate line
+    let edition = toml.lines().position(|line| line.trim() == "edition = \"2018\"");
+    if edition.is_none() {
```

Comment:
Rather than using an option here, we can do `lines().any(...)` and that'll return a bool

Reply:
Done.